swagger_server/controllers/itm_ta2_eval_controller.py

```diff
@@ -108,18 +108,20 @@ def _reclaim_old_session():
             session_mapping.pop(session_id)             # From both places
             return ITMSession()
 
-def start_session(adm_name, session_type, kdma_training=None, max_scenarios=None):  # noqa: E501
+def start_session(adm_name, session_type, adm_profile=None, kdma_training=False, max_scenarios=None):  # noqa: E501
     """Start a new session
 
     Get unique session id for grouping answers from a collection of scenarios/probes together # noqa: E501
 
-    :param adm_name: A self-assigned ADM name.  Can add authentication later.
+    :param adm_name: A self-assigned ADM name.
     :type adm_name: str
-    :param session_type: the type of session to start (&#x60;test&#x60;, &#x60;eval&#x60;, or a TA1 name)
+    :param session_type: the type of session to start `test`, `eval`, or a TA1 name)
```

Comment:
Really small thing, but it looks like a parenthesis was lost here

Reply:
I will fix this in my upcoming PR for ITM-227, thanks.